jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskcomments/TaskCommentsPresenterTest.java

```diff
@@ -133,4 +135,56 @@ public void removeCommentAdded() {
                                                     anyLong());
         verify(viewMock).redrawDataGrid();
     }
+
+    @Test
+    public void taskSelectionEventIsForLogTask() {
+        String serverTemplateId = "serverTemplateId";
+        String containerId = "containerId";
+        Long taskId = 1L;
+        boolean isForLog = true;
+        TaskSelectionEvent event = new TaskSelectionEvent(serverTemplateId,
+                                                          containerId,
+                                                          taskId,
+                                                          "task",
+                                                          true,
+                                                          isForLog);
+        CommentSummary comment1 = new CommentSummary(1,
+                                                     "commentText",
+                                                     "ByTest",
+                                                     new Date());
+        when(commentsServiceMock.getTaskComments(eq(serverTemplateId),
+                                                 eq(containerId),
+                                                 eq(taskId))).thenReturn(Arrays.asList(comment1));
+
+        presenter.onTaskSelectionEvent(event);
+
+        verifyNoMoreInteractions(commentsServiceMock);
```

Comment:
if all you want to check is that the `getTaskComments` isn't called, I would recommend using `verifyZeroInteractions` instead and perhaps even not do any mock preparation with `when`.